composer.json

```diff
@@ -20,11 +20,11 @@
         "symfony/framework-bundle": "^3.4.34|^4.3|^5.4|^6.0",
         "symfony/security-bundle": "^3.4.34|^4.3|^5.4|^6.0",
         "symfony/validator": "^3.4.34|^4.3|^5.4|^6.0",
-        "paysera/lib-normalization-bundle": "^1.1.0",
+        "paysera/lib-normalization-bundle": "^1.1",
```

Comment:
Changing from `^1.1.0` to `^1.1` doesn't appear to alter the library's version constraints in any meaningful way. Both forms allow the same range of versions (from `1.1.0` upwards, but less than `2.0.0`).

It looks like you might have confused this with the `~` operator, which behaves differently. If the operator had been `~`, changing from `~1.1.0` to `~1.1` would have introduced actual changes in the version constraints. Feel free to close this PR if this was the case.